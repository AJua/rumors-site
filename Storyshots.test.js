import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { createMount, createRender } from '@material-ui/core/test-utils';
import initStoryshots from '@storybook/addon-storyshots';

Enzyme.configure({ adapter: new Adapter() });

initStoryshots({
  renderer: createMount(),
  // snapshotSerializers: [createRender()],
});
