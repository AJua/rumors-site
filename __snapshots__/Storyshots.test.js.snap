// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Example story Hello World 1`] = `
<div>
  Hello world!
</div>
`;
